tests/sentry/api/endpoints/test_sentry_app_components.py

```diff
@@ -134,14 +132,14 @@ def test_project_not_owned_by_org(self, run):
         assert response.status_code == 404
         assert response.data == []
 
-    @patch("sentry.mediators.sentry_app_components.Preparer.run")
+    @patch("sentry.sentry_apps.SentryAppComponentPreparer.run")
```

Comment:
Mocking the application logic an endpoint uses makes me suspicious of how useful these tests actually are. We can revisit that later though.

Reply:
don't disagree*